modules/javafx.graphics/src/main/java/com/sun/glass/ui/gtk/GtkWindow.java

```diff
@@ -48,7 +48,7 @@ public GtkWindow(Window owner, Screen screen, int styleMask) {
 
     // empty - not needed by this implementation
     @Override
-    protected void _updateViewSize(long ptr) {}
+    protected native void _updateViewSize(long ptr);
```

Comment:
Is there a system test that can be used to verify these changes on Linux? Since you have mentioned this is the same as #1054, I take that SetSceneScalingTest can be used. 

Also, I'm not sure how you would run WinIconified test attached to the JBS ticket to see that it fails or passes. If that can be converted to system test, could you add it to this PR?

Reply:
I noticed this earlier, and filed [JDK-8298500](https://bugs.openjdk.org/browse/JDK-8298500) to improve our test coverage in this area.

It would be good to make sure that we have a test that does this for at least an inconified window to help validate this PR.

Reply:
I could do a Iconified test, but I think it would not test the case that happened before, the window pops (because of `requestFocus`) and then gets iconified (the user sees the window showing then iconifying).

Reply:
That seems good enough to verify that the window doesn't stay showing.

Reply:
@kevinrushforth I was checking and `SceneChangeShouldNotFocusStageTest` added on [JDK-8296621](https://bugs.openjdk.org/browse/JDK-8296621) does exactly that.

But I noticed the same as you described on [JDK-8298500](https://bugs.openjdk.org/browse/JDK-8298500).

@jperedadnr The `WinIconified.java` attached is to show that a window should stay iconified and that it is initially iconified (does not pop on the screen and then gets iconified).

Reply:
`SceneChangeShouldNotFocusStageTest` covers the test that the window should start iconified. It does not cover the test where the window pops and then iconifies (causing a visual "weirdness"). I don't think the later is possible on a systemTest.

Reply:
If not as system test, then could you add `WinIconified` as manual test, with some javadoc as quick instructions on how to check that it works as expected?

Reply:
@jperedadnr added.

Reply:
Thanks, @tsayao 

I'm running these tests now on Linux (Ubuntu 20.04) and MacOS (13.3.1):

- controls:test
- Manual/ StartIconified
- System/Robot test.robot.javafx.stage.IconifyTest
- System/Robot test.robot.javafx.scene.SceneChangeShouldNotFocusStageTest

On Linux, all of them work fine.
On MacOS:
 - Work fine
 - I see the stage, it is not iconified (with or without your patch). Is this a know issue?
 - Works fine
 - Fails: it is not iconified (with or without your patch, also I had another issue with scene being null). Is it a known issue as well?

Reply:
I don't have a mac to test it. Was it working before?

Reply:
Hmm, tested 17 to 21-ea, it doesn't work (StartIconified always shows the stage on macOS)